src/ucp/core/ucp_worker.h

```diff
@@ -168,7 +169,10 @@ typedef struct ucp_worker {
     ucs_list_link_t               stream_eps;    /* List of EPs with received stream data */
     khash_t(ucp_worker_ep_hash)   ep_hash;       /* Hash table of all endpoints */
     khash_t(ucp_ep_errh_hash)     ep_errh_hash;  /* Hash table of error handlers associated with endpoints */
-    ucp_worker_iface_t            *ifaces;       /* Array of interfaces, one for each resource */
+    ucs_list_link_t               ifaces;        /* List of all interfaces (device and sockaddr),
```

Comment:
comment 'one for each resource' seems to be confusing.  We may have sockaddr ifaces for some of ifaces only, right?

Reply:
Yes, will remove this line.